mitodl/open-discussions
# Third pass of email templates

#### Pre-Flight checklist


#### What are the relevant tickets?
#1325 

#### What's this PR do?
Updates email templates and fixes a bug where `{{ site_name }}` is not being set in the email context. Also updated quite a few tests to use the default response data stuff @gsidebo wrote to cut down on the amount of code.

#### How should this be manually tested?
Go to `/__emaildebugger__/` and test render the frontpage email and verify it matches the mockup (sans thumbnail images which we're still skipping)

#### Screenshots
![screenshot_2018-10-12 email debugger](https://user-images.githubusercontent.com/28598/46879234-71939d00-ce13-11e8-889a-53e47a97c044.png)
